zkevm-circuits/src/evm_circuit/witness.rs

```diff
@@ -1074,7 +1074,7 @@ impl From<&circuit_input_builder::ExecStep> for ExecutionState {
                 }
                 match op {
                     OpcodeId::ADD => ExecutionState::ADD,
-                    OpcodeId::MUL => ExecutionState::MUL,
+                    OpcodeId::MUL | OpcodeId::DIV | OpcodeId::MOD => ExecutionState::MUL,
```

Comment:
```suggestion
                    OpcodeId::MUL | OpcodeId::DIV | OpcodeId::MOD => ExecutionState::MUL_DIV_MOD,
```

Maybe this to fit in with the rest of naming?

Reply:
I remember I had a similar suggestion for `ADD` -> `ADD_SUB`. @han0110 thought it's a bit confusing. So what do you think about the execution state renaming?

Reply:
If it's more clear in this way, we should just rename it. My thought was to not create new term, but it seems inevitable.

Reply:
Sure. I'll make the change.